src/tools/fuzzing.h

```diff
@@ -1200,20 +1200,23 @@ class TranslateToFuzzReader {
   Expression* makeCall(Type type) {
     // seems ok, go on
     int tries = TRIES;
+    bool isReturn;
     while (tries-- > 0) {
       Function* target = func;
       if (!wasm.functions.empty() && !oneIn(wasm.functions.size())) {
         target = vectorPick(wasm.functions).get();
       }
-      if (target->result != type) {
+      isReturn = type == unreachable && wasm.features.hasTailCall() &&
+                 func->result == target->result;
```

Comment:
shouldn't this also have a random factor, and not do this all the time?

Reply:
Previously `makeCall{Indirect}(unreachable)` would never actually return a call, since no function returns `unreachable`. So adding randomness here would just make `makeCall` return a non-call operand sometimes, which doesn't seem useful.